File: Tutorial3SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/local/MeasurementBubbleLevelDao.java
package com.example.tutorial3superclassentities.data.source.local;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Query;

import com.example.tutorial3superclassentities.data.source.model.MeasurementBubbleLevel;

import java.util.List;

@Dao
public interface MeasurementBubbleLevelDao extends BaseDao<MeasurementBubbleLevel> {
    @Query("SELECT * FROM table_clinometer_bubble_level")
    List<MeasurementBubbleLevel> getAll();

    @Query("DELETE  FROM table_clinometer_bubble_level")
    int deleteAll();
}

File: Tutorial3-1SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/local/MeasurementBubbleLevelDao.kt
package com.example.tutorial3superclassentities.data.source.local

import androidx.room.Dao
import androidx.room.Query
import com.example.tutorial3superclassentities.data.source.model.MeasurementBubbleLevel

@Dao
interface MeasurementBubbleLevelDao : BaseDao<MeasurementBubbleLevel> {

    @Query("SELECT * FROM table_clinometer_bubble_level")
    fun getAll(): List<MeasurementBubbleLevel>

    @Query("DELETE  FROM table_clinometer_bubble_level")
    fun deleteAll(): Int
}